### PR TITLE
Resend invitation mails after a few days

### PR DESCRIPTION
**Pull request by gmethvin**

### CODE REVIEW

File: shoebox/app/com/keepit/common/mail/InvitationMailPlugin.scala [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  def receive = {
+    case NotifyAcceptedUser(userId) =>
+      sendNotification(userId,
+        "Congrats! You're in the KiFi Private Beta",
+        views.html.email.invitationAccept(_).body)
+    case ResendNotifications =>
+      log.debug("Checking for invitations to resend")
+      for {
+        inv <- db.readOnly { implicit s => invitationRepo.getAdminAccepted() }
+        userId <- db.readOnly { implicit s => suiRepo.get(inv.recipientSocialUserId).userId }
```
Comment (eishay) [inline shoebox/app/com/keepit/common/mail/InvitationMailPlugin.scala]: the `db.readOnly` calls are called sequentially, why not reuse the readonly session for the two calls?

File: shoebox/test/com/keepit/common/mail/InvitationMailPluginTest.scala [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+import com.keepit.inject.inject
+import com.keepit.model._
+import com.keepit.test.{FakeClock, EmptyApplication, DbRepos}
+
+import akka.actor.ActorSystem
+import akka.testkit.TestKit
+import play.api.Play.current
+import play.api.test.Helpers.running
+
+class InvitationMailPluginTest extends TestKit(ActorSystem()) with Specification with DbRepos {
```
Comment (eishay) [inline shoebox/test/com/keepit/common/mail/InvitationMailPluginTest.scala:19]: nice test!

File: shoebox/test/com/keepit/test/TestApplication.scala [hunk 3]
```diff
@@ -198,6 +194,16 @@ class FakeClock extends Clock with Logging {
     }
   }
 
+  def +=(p: ReadablePeriod) {
```
Comment (eishay) [inline shoebox/test/com/keepit/test/TestApplication.scala:197]: nice

File: shoebox/app/com/keepit/common/mail/InvitationMailPlugin.scala (hunk 1 above)
Comment (gmethvin) [inline shoebox/app/com/keepit/common/mail/InvitationMailPlugin.scala]: Might be a good idea.

File: shoebox/app/views/email/invitationReminder.scala.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
+@(user: User)
```
Comment (andrewconner) [inline shoebox/app/views/email/invitationReminder.scala.html:1]: Is the wording different on this one?

Comment (gmethvin) [inline shoebox/app/views/email/invitationReminder.scala.html:1]: It's slightly different now.

Comment (gmethvin) [inline shoebox/app/views/email/invitationReminder.scala.html:1]: The main difference is `This is a reminder that you were approved to join the KiFi beta group`, per Effi's request.

**Pull request merged**